packages/f/file2arraybuffer.json

```
@@ -0,0 +1,37 @@
+{
+  "name": "file2arraybuffer",
+  "filename": "file-to-arraybuffer.min.js",
+  "homepage": "https://github.com/juliolmuller/file2arraybuffer-js",
+  "description": "Straight-forward function to convert files into ArrayBuffer objects.",
+  "license": "MIT",
+  "authors": [
+    {
+      "name": "Julio L. Muller"
+    }
+  ],
+  "keywords": [
+    "file",
+    "blob",
+    "binary",
+    "buffer",
+    "arraybuffer",
+    "attachment",
+    "upload"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/juliolmuller/file2arraybuffer-js"
+  },
+  "autoupdate": {
+    "source": "npm",
+    "target": "file2arraybuffer",
+    "fileMap": [
+      {
+        "basePath": "dist",
+        "files": [
+          "**/*"
```

```suggestion
          "*.js"
```
We don't accept open globstars in configs as it can lead to files being included on the CDN that we don't expect. It looks like there are just a couple of JS files in the dist directory that are needed?